tsl/src/bgw_policy/job.c

```diff
@@ -622,6 +622,63 @@ policy_recompression_execute(int32 job_id, Jsonb *config)
 	return true;
 }
 
+void
+policy_move_hyper_inval_read_and_validate_config(Jsonb *config,
```

Comment:
Maybe update these function names in line with "process_hypertable_invalidation" function in underlying PR?

```suggestion
policy_process_hyper_inval_read_and_validate_config(Jsonb *config,
```